server/utils/users/homePageDashboard.ts

```diff
@@ -98,10 +107,14 @@ export const sectionsForUser = (user: UserDetails): Array<ServiceSection> => {
     items.push(sections.assess)
   }
 
-  if (managerRoles.some(role => hasRole(user, role))) {
+  if (hasRole(user, 'manager')) {
```

Comment:
Given the tiles are identical I could see this being confusing for users who have both 'manager' and 'future_manager' roles (which shouldn't happen but will). I wonder if we could either have some differentiation in the copy and/or title *or* use an `if...else` to ensure they only get one 'manage' tile

Reply:
yes good point, I did think about this but wasn't sure which way to go...

On reflection I think that we should show the v2 tile if the user has the future manager role. I'll tweak this...